sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/InMemoryCatalog.scala

```diff
@@ -346,6 +346,13 @@ class InMemoryCatalog(
     StringUtils.filterPattern(listTables(db), pattern)
   }
 
+  override def listViews(db: String, pattern: String): Seq[String] = synchronized {
+    requireDbExists(db)
+    val views = catalog(db).tables.filter(v => v._2.table.tableType == CatalogTableType.VIEW)
```

Comment:
nit: `.filter(_._2.table.tableType == CatalogTableType.VIEW)`

Reply:
Thanks, updated in c5570d249426a821896ab85e37bf2e0c1d9a39ee.